src/utils/serializer/assignments.serializer.ts

```diff
@@ -0,0 +1,23 @@
+import { Assignment } from 'devu-shared-modules'
+
+import AssignmentModel from '../../model/assignments.model'
+
+export function serialize(assignment: AssignmentModel): Assignment {
+  return {
+    id: assignment.id,
+    courseId: assignment.courseId,
+    name: assignment.name,
+    startDate: assignment.startDate,
+    dueDate: assignment.dueDate,
+    endDate: assignment.endDate,
```

Comment:
I think we should call `toISOString` here to be consistent. For all I know without it it might look the same anyway, but to be explicit and consistent, we should add it.